src/JsonSchema/ConstraintError.php

```diff
@@ -93,7 +93,7 @@ public function getMessage()
             self::NOT => 'Matched a schema which it should not',
             self::ONE_OF => 'Failed to match exactly one schema',
             self::REQUIRED => 'The property %s is required',
-            self::REQUIRED_D3 => 'Is missing and it is required',
+            self::REQUIRED_D3 => 'The property %s is required',
```

Comment:
This line is redundant - there's already an identical message template on line 95 (`REQUIRED`), so IMO it makes more sense to just use that.